news/2019-12-04-new-featured-artist-kurokotei.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "New Featured Artist: Kurokotei"
+date: 2019-12-04 10:00:00 +0000
+---
+
+**Kurokotei** soars in from a collapsed galaxy to join the Featured Artist lineup!
+
+![](https://assets.ppy.sh/artists/58/header.jpg)
+
+We're super proud to announce that **Kurokotei** is the latest name to join the ever-expanding Featured Artist roster, complete with a selection from her latest album, *Into the Void*. As an extra bonus, **Kurokotei** is also a long-time member of the osu!community, an established mapper and top-end osu!taiko player in her own right. There's something extra special about bringing people aboard from the community!
+
+The older among us might remember **Kuroktei**'s track titled *Galaxy Collapse*, made famous in osu! by [the now Loved set](https://osu.ppy.sh/beatmapsets/396221#osu/862088). It remains to this day, one of the hardest beatmaps ever made, and we all know just how much the osu! community loves its ridiculously difficult maps.
+
+We're confident that some of **Kurokotei**'s latest work has even more of that glorious, glorious spark to kindle some extra passionate creations, and we're excited to see what people make with it all. Don't be intimidated by the history if you're new to mapping - **Kurokotei**'s work is more than suitable for anyone with enough spirit to tackle it, so be sure to give it a go.
```

Comment:
```suggestion
We're confident that some of **Kurokotei**'s latest work has even more of that glorious, glorious spark to kindle some extra passionate creations, and we're excited to see what people make with it all. Don't be intimidated by the history if you're new to mapping—**Kurokotei**'s work is more than suitable for anyone with enough spirit to tackle it, so be sure to give it a go.
```
u knew this would happen 😠